content/plugins/1.0/installation.md

```diff
@@ -10,18 +10,28 @@ menu: "plugins-1.0"
 
 ### Gems
 
-Installing via gem is the preferred method if one exists.
-
-For production deployments make sure you [pin your gems](https://github.com/sensu-plugins/community/blob/master/best_practices/production_deployments/plugins/PINNING_VERSIONS.md)
+Installing via the built-in `sensu-install` is the preferred method of installing a plugin.
 
 **Standard Installation**
 
 ```
-gem install sensu-plugins-disk-checks
+sensu-install -p disk-checks
 ```
 
+For further details, please see the [plugin installation guide][1].
+
+### Gems
```

Comment:
This heading seems like a duplicate of the one in line 11. Looks like maybe the heading in line 11 should be removed?